curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.english.md

```diff
@@ -24,6 +24,7 @@ json.forEach(function(val) {
 });
 ```
 
+<strong>Note:</strong> For this challenge you need to add new HTML elements to the page, so you cannot rely on `textContent`. Instead you need to use `innerHTML`, which can make a site vulnerable to cross site scripting attacks.
```

Comment:
```suggestion
<strong>Note:</strong> For this challenge, you need to add new HTML elements to the page, so you cannot rely on `textContent`. Instead you need to use `innerHTML`, which can make a site vulnerable to Cross-site scripting attacks.
```

Reply:
@Manish-Giri Can you also add a comma after the word `Instead` in the second sentence?

Reply:
This is not my PR, and I cannot commit this suggestion via the interface. I made the original suggested changes on behalf of the OP in hopes to get this PR merged and I think it's fine how it is. If you want it this way, you should make the changes.

Reply:
The suggestions @Manish-Giri made will improve the challenge.

Reply:
@Manish-Giri I have added a new commit with your suggested changes.  If you are OK with the PR now, can you approve and merge it?  Thanks.